quran/quran.com-frontend-next
# Add verse page

### Summary
This PR adds an incrementally static generated page where the user can navigate to a specific verse within a chapter. A specific verse can be accessed through navigation using `{chapterNumber}/{verseNumber}` path in the url e.g. {url}/2/6 which navigates to the 6th verse of chapter 2.

### Test Plan

1. Navigate to the 1st chapter through {url}/1.
2. The list of the verses should show and on the left side of each verse there should be a clickable section with the following format `{chapterNumber}:{verseNumber}` e.g. `1:3`.
3. Click on the section from step 2 for one of verses let's say the 3rd verse.
4. You will be navigated to a page that only shows the 3rd verse.
5. You can change the translations/font from the Developer Utility and it should reflect instantly.
6. Click on `Go Back` section on the top left side of the verse's page which should take you back to the verse's chapter.

### Screenshots

| Before | After |
| ------ | ------ |
| `{url}/1` <img width="1440" alt="Before" src="https://user-images.githubusercontent.com/15169499/126103808-d46cb82b-3fd4-4b9e-b461-df1af404ee8b.png"> | `{url}/1` <img width="1440" alt="After 1" src="https://user-images.githubusercontent.com/15169499/126103794-056050b8-e2b2-4d98-b838-5b660eb01c38.png"> |
|  |`{url}/1/3`<img width="1440" alt="After 2" src="https://user-images.githubusercontent.com/15169499/126103806-e6e03a27-a1d5-4bb9-b7d5-07a446b57f27.png"> |
